src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Http/Http.ts

```diff
@@ -267,15 +267,16 @@ export class Service<R extends ResourcesBase.IResource> {
     }
 
     public delete(path : string, contentType : string, config : IHttpConfig = {}) : angular.IPromise<any> {
-        var obj = {
-            content_type: contentType,
-            data: {}
-        };
-        obj.data[SIMetadata.nick] = {
-            deleted: true
-        };
+        if (this.adhPreliminaryNames.isPreliminary(path)) {
+            throw "attempt to http-put preliminary path: " + path;
```

Comment:
why does this constitute a an attempt to http-*put*, and not an attempt to http-*delete*?